diff -ruN -x mobiquo -x cache -x files -x images -x styles -x docs -x ext -x install -x .install -x SPRING.patch phpBB3/config.php phpbb/config.php
--- phpBB3/config.php	2015-02-01 19:23:21.000000000 +0100
+++ phpbb/config.php	2015-03-09 18:11:50.983438306 +0100
@@ -0,0 +1,20 @@
+<?php
+// phpBB 3.1.x auto-generated configuration file
+// Do not change anything in this file!
+include(dirname(__FILE__) . '/../../springpw.php');
+$dbhost = $spring_dbhost;
+$dbname = $spring_dbname;
+$dbuser = $spring_dbuser;
+$dbpasswd = $spring_dbpass;
+
+$dbms = 'phpbb\\db\\driver\\mysqli';
+$dbport = '';
+$table_prefix = 'phpbb3_';
+$phpbb_adm_relative_path = 'adm/';
+$acm_type = 'phpbb\\cache\\driver\\file';
+
+@define('PHPBB_INSTALLED', true);
+// @define('PHPBB_DISPLAY_LOAD_TIME', true);
+// @define('DEBUG', true);
+// @define('DEBUG_CONTAINER', true);
+
